models/classes/import/QtiTestImporter.php

```diff
@@ -35,33 +38,37 @@ class QtiTestImporter extends AbstractTestImporter
 
     /**
      * @param File $file
-     * @param \core_kernel_classes_Class $class
+     * @param core_kernel_classes_Class|null $class
      * @param bool $enableMetadataGuardians
      * @param bool $enableValidators
      * @param bool $itemMustExist
      * @param bool $itemMustBeOverwritten
-     * @return \common_report_Report
+     * @param bool $overwriteTest
+     * @param string|null $itemClassUri
+     * @param string|null $overwriteTestUri
+     * @return common_report_Report
      */
     public function import(
         File $file,
-        \core_kernel_classes_Class $class = null,
-        $enableMetadataGuardians = true,
-        $enableValidators = true,
-        $itemMustExist = false,
-        $itemMustBeOverwritten = false,
+        core_kernel_classes_Class $class = null,
+        bool $enableMetadataGuardians = true,
+        bool $enableValidators = true,
+        bool $itemMustExist = false,
+        bool $itemMustBeOverwritten = false,
```

Comment:
Are we sure that all places pass proper booleans here? If someone is passing 0 or 1 instead or any other invalid value, we would be breaking compatibility.